docs/shins/index.html

```diff
@@ -2185,7 +2185,7 @@ <h3 id="fetch-authorization-token-responses">Responses</h3>
 <tr>
 <td>default</td>
 <td>Default</td>
-<td>Possible error responses. Errors can be from both Vipps' own code (where we have full control) and from Microsoft Azure (where we rely on standard functionality). See the <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md">&quot;Common errors&quot; section of the FAQ</a>. See also: <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation">Responses</a>.</td>
+<td>Possible error responses. See <a href="https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation">Responses</a>.</td>
```

Comment:
Same here: The `ErrorFromAzure` was a nice improvement, so I'd like to keep that.